enterprise/management/commands/email_drip_for_missing_dsc_records.py

```diff
@@ -176,9 +176,15 @@ def handle(self, *args, **options):
                 enterprise_customer=enterprise_customer
             )
             course_accessed = False
-            if is_course_accessed and is_course_accessed(ec_user.user, course_id):
-                course_accessed = True
-
+            try:
+                if is_course_accessed and is_course_accessed(ec_user.user, course_id):
+                    course_accessed = True
+            except Exception as exc:  # pylint: disable=broad-except
+                LOGGER.exception('[Absent DSC Email] Error in {course} for user {user}. Error detail: {exc}'.format(
+                    course=course_id,
+                    user=username,
+                    exc=str(exc)
```

Comment:
@irfanuddinahmad Exception information is automatically added when we use `logging.exception` so we don't need add `exc` in log.